recipe/schemas.py

```diff
@@ -496,6 +496,38 @@ def _replace_references(shelf):
                         quickfilter_schema
                 },
             ),
+        'BucketDimension':
+            S.Dict(
+                allow_unknown=True,
+                coerce=_move_extra_fields,
+                schema={
+                    'field':
+                        'non_aggregated_field',
+                    'buckets':
+                        S.List(
+                            schema=S.Dict(
+                                schema={
+                                    # FIXME should allow any value?
+                                    'value': S.String(),
+                                    'condition': S.Dict(
+                                        schema={
+                                            'between': S.List(
+                                                schema=S.Number()
```

Comment:
I tried to create stricter schema (list of **2** numbers). Seems you can use `items` in cerberus (http://docs.python-cerberus.org/en/stable/validation-rules.html#items) but not sure how to do it with sureberus.